backend/console_proxy.pm

```diff
@@ -52,7 +52,7 @@ sub AUTOLOAD {
 			    args => $args,
 			   };
 
-	bmwqemu::log_call($function, wrapped_call => $wrapped_call);
```

Comment:
no, this one does not count because this is old code, you didn't write it which should normally mean: you would do it different today ;-P